protocol-definitions/DynamicConfig.yaml

```
@@ -1246,11 +1246,11 @@ methods:
             apply to this lock configuration's operations.
     response: {}
   - id: 17
-    name: addExternalDataStoreConfig
-    since: 2.5
+    name: addDataLinkConfig
+    since: 2.6
     doc: |
-      Adds an external data store configuration.
-      If an external data store configuration with the given {@code name} already exists, then
+      Adds a data link configuration.
+      If an data link configuration with the given {@code name} already exists, then
```

nit: feel free to ignore :)

```suggestion
      If a data link configuration with the given {@code name} already exists, then
```